integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -94,7 +94,7 @@
   public static final int MAX_NAME_LENGTH = 255;
 
   private static InstancedConfiguration sConf =
-      new InstancedConfiguration(ConfigurationUtils.defaults());
+          new InstancedConfiguration(ConfigurationUtils.defaults());
```

Comment:
@iluoeli we are using 4-space indent rather than 8. Checkout https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#coding-style for the Eclipse Code Formatter or Eclipse Code Formatter Plugin
Let's avoid introducing unnecessary change in a PR which increases the footprint and slows down code review

Reply:
actually for this file, let's simple leave it unchanged as what is in master branch.